builder/tests/builder/src/test/java/io/helidon/builder/test/AnnotationCaseTest.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2022, 2023 Oracle and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.helidon.builder.test;
-
-import java.util.Arrays;
-
-import io.helidon.builder.test.testsubjects.AnnotationCase;
-import io.helidon.builder.test.testsubjects.AnnotationCaseExtDefault;
-
-import org.junit.jupiter.api.Test;
-
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.sameInstance;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.contains;
-
-class AnnotationCaseTest {
-
-    @Test
-    void testIt() {
```

Comment:
this type of test should still exist somewhere

Reply:
This is not supported now. If we add support for this kind of prototypes, we should re-add the tests as well.